README.md

```diff
@@ -1,25 +1,25 @@
 # Werewolf-Brain
 **Werewolf-Brain** (or **wwb**) is a JavaScript library 
-for creating scenarios for playing the party card game 
+for the creation of *scenarios* to play the party card game 
 [Ultimate Werewolf](https://en.wikipedia.org/wiki/Ultimate_Werewolf "Wikipedia").
-There are several ways to craft funny games with this 
-library, check out the examples section.
+There are several ways to craft fun games with this 
+library. For more on usage, check out the examples section below.
 
 ## Installing
 If you use npm, ```npm install werewolf-brain```. Otherwise, 
 download the files from [here](https://github.com/wistcc/werewolf-brain "Github") and add them to the project.
 
 ## What does it do?
-Creates new scenarios based on the number of players, on a 
-template, or the available cards. The number of players is not 
-limited. If the current deck of cards can't have a game with all
-the players, the library will give you a game with the maximum 
-number of players that could found in 5000 tries. 
-
-We use a game object for describing a scenario. This object contains 
-the number of players, the game weight and the deck itself. The game 
+Creates new scenarios based on the number of players,  or available cards, given a template. 
+Moderators can then use these scenarios to set up Werewolf games. 
+The number of players is not limited. If the current deck of cards 
+can't have a game with all the players, the library will give you a
+game with the maximum number of players that it could calculate in 5000 tries. 
+
+We use a **game** object to describe a scenario. This object contains 
+the number of *players*, the game *weight* and the *deck* itself. The game 
 weight gives you an idea of how balanced is the game. Positive weight 
-means a game in favor to the villagers, negatives means the opposite.
+means a game in favor to the Villagers, negatives means the opposite: a game in favor to the Werewolves.
```

Comment:
Although I completely understand what you are saying, and although I would still debate the usage of 'the opposite', I suggest to use a more expressive approach: Just as I was confused about the notion of 'the opposite', others may as well be. (Which is exactly why I wanted to add another level of explanation to the 'opposite' proposition).

In the original sentence, if we remove the 'negatives means the opposite' part, the sentence would retain its meaning. On the other hand, if we leave it as is, it could be open to the interpretation of the reader, who might or might not do the same mistake I did, the mistake of assuming that the other roles would necessarily be Werewolves (which is prone to happen if the reader is not an experienced Ultimate Werewolf gamer). If we decide to go with this line of thinking, a "solution" would be to expand a bit further on what 'the opposite' is, without being redundant. In this case, I **guess** we could said:

`means a game in favor to the Villagers, negative means the opposite: a game in favor to the other non-Villagers cards`

As a further note, and just to add: Although arguable, I (as in **my** opinion) also have some issues with using 'the opposite' to * simply* denote what something is NOT, as to me it sounds like a double negative (if you have multiple choices and you select one, would that choice be the opposite of all the other ones? As long as the others choices are different, the others will always be no). I would use 'the opposite' as the enter point to present an opposite possibility, which would explain what 'the opposite' would be.